convert_kinbody_to_urdf.py

```diff
@@ -0,0 +1,194 @@
+# convert_kinbody_to_urdf.py
+
+import os
+import sys
+import xml.etree.ElementTree as ET
+
+
+class Kinbody_Converter:
+    def __init__(self):
+        self.kinbody = None
+        self.urdf = None
+        self.link = None
```

Comment:
I checked URDFs by drawing them on RViz by using pantry-loading but did not directly compare them with `kinbody` loaded. I will double check and update.